apps/main/src/store/createPoolListSlice.ts

```diff
@@ -195,14 +194,14 @@ const createPoolListSlice = (set: SetState<State>, get: GetState<State>) => ({
       userPoolList: UserPoolListMapper | undefined
     ) => {
       const activeKey = getPoolListActiveKey(rChainId, searchParams)
-      const storedFormValues = get()[sliceKey].formValues
-      const storedResults = get()[sliceKey].result[activeKey]
+      const state = get()[sliceKey]
```

Comment:
Is there a good reason to call `get()[sliceKey]` repeatedly?

Reply:
Depends on where it is being called, if it is used after an await, then the state can be state, or else I don't think it is necessary. I also did not measure, so not sure if it is causing a memory issue.